src/monitor/main_monitor.cpp

```diff
@@ -41,6 +41,8 @@ MainMonitor::MainMonitor()
     // the first possible timestamp in the trace
     trace_.begin_record();
 
+    trace_.archive().set_creator(config().command_line);
```

Comment:
The creator should be lo2s with some version information. See #39. IMHO `set_description()` fits better here. If it does not exist in otf2xx yet, then make a PR there as well, please.